src/python/pants/core/goals/fmt.py

```diff
@@ -70,13 +81,15 @@ def message(self) -> str | None:
         if self.skipped:
             return f"{self.formatter_name} skipped."
         message = "made changes." if self.did_change else "made no changes."
-        output = ""
-        if self.stdout:
-            output += f"\n{self.stdout}"
-        if self.stderr:
-            output += f"\n{self.stderr}"
-        if output:
-            output = f"{output.rstrip()}\n\n"
+
+        if self.did_change:
+            output = "".join(
+                f"\n  {file}"
+                for file in SnapshotDiff.from_snapshots(self.input, self.output).changed_files
+            )
```

Comment:
I kinda want to have this take an optional attribute (`additional_output`) which on `lint` would be provided `Run ./pants fmt`. We ran into an issue where the `lint` output confused someone because it isn't obvious `black` or any other formatter is a _formatter_.

Thoughts?

Reply:
Although ideally it'd be nice to have this output near the "summary" as part of the goal output at the very bottom.

Reply:
That would work, but would be easier to do once at the end of the `lint` goal probably, rather than as part of the streaming output?